spec/index.html

```diff
@@ -4648,7 +4648,7 @@ <h3>Operand Data Types</h3>
           <code>literals</code> passed as arguments to these functions and operators are mapped
           to XML Schema typed values with a <a data-cite="XPATH20#dt-string-value">string value</a> of
           the <code>lexical form</code> and an 
-          <a href="http://www.w3.org/TR/xmlschema-2/#dt-atomic">atomic datatype</a> corresponding to the
+          <a data-cite="XMLSCHEMA-2#dt-atomic">atomic datatype</a> corresponding to the
```

Comment:
`XMLSCHEMA-2` is XML schema 1.0.

```suggestion
          <a data-cite="XMLSCHEMA11-2#dt-atomic">atomic datatype</a> corresponding to the
```
which goes to http://www.w3.org/TR/xmlschema11-2#dt-atomic